src/components/targets/createTarget/selectDropdown.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from "react";
+import 'components/targets/createTarget/selectDropdown.css'
+
+import football from 'assets/images/football.png'
+import music from 'assets/images/music.png'
+import food from 'assets/images/food.png'
+import travel from 'assets/images/travel.png'
+
+interface SelectDropDownProps {
+  options: { label: string; value: string }[];
+}
+
+interface ImgHash {
+  [key: string]: string;
+}
+
+const imgHash: ImgHash = {
+  'football': football,
+  'music': music,
+  'food': food,
+  'travel': travel
+}
+
+const SelectDropDown = ({ options, ...props }: SelectDropDownProps) => {
+  const handleSelectChange = (event: any) => {
+    const option = event.target.textContent;
+    let selectPlaceholder = document.querySelectorAll('.selectdropdown--placeholder')[0];
+    selectPlaceholder.textContent = option;
+  };
+
+  const changeMenuVisibility = (event: any) => {
+    let visibility  = event.target.parentElement.nextSibling.style.visibility;
+    const menu = event.target.parentElement.nextSibling;
+
+    if(visibility === 'visible') {
+      menu.style.visibility = 'hidden'
+    }else{
+      menu.style.visibility = 'visible'
+    }
+  }
+
```

Comment:
WDYT about handling your dropdown visibility with a piece of state?
Something like
`const [areOptionsVisible, setAreOptionsVisible] = useState(false)`

```suggestion
  const toggleOptionsVisibility = () => {
       setAreOptionsVisible(prevValue => !prevValue)
   }

```